server/ansible/roles/pbench-server-activate-create-crontab/tasks/main.yml

```diff
@@ -0,0 +1,6 @@
+---
+- name: create crontab
+  command: pbench-server-activate-create-crontab /opt/pbench-server/lib/crontab
```

Comment:
Will the crontab always be at `/opt/pbench-server/lib/crontab`?  Is the install directory configurable?

Reply:
I''l replace all the `/opt/pbench-server` instances with an {{ install_dir }} variable.